filebeat/tests/system/test_registrar.py

```diff
@@ -194,3 +195,76 @@ def test_rotating_file(self):
 
         # Check that 2 files are port of the registrar file
         assert len(data) == 2
+
+
+    def test_rotating_file_inode(self):
+        """
+        Check that inodes are properly written during file rotation
+        """
+        self.render_config_template(
+            path=os.path.abspath(self.working_dir) + "/log/input*",
+            scan_frequency="1s"
+        )
+
+        if os.name == "nt":
```

Comment:
Saw this one, but it seems like st_ino inode is also available in SunOS, only st_dev is special.